packages/storybook/src/theme.ts

```diff
@@ -33,6 +33,7 @@ const colors = {
   danger: baseColors.red[2],
   muted: baseColors.gray[3],
   highlight: baseColors.blue[3],
+  muted: baseColors.gray[3],
```

Comment:
I think I added muted also so we can remove one of the two

Reply:
Yep, thanks!